build.sbt

```diff
@@ -279,7 +304,9 @@ lazy val jsonPlay = crossProject(JVMPlatform, JSPlatform)
     libraryDependencies ++= List(
       "com.typesafe.play" %%% "play-json" % playVersion,
       "org.gnieh" %%% "diffson-play-json" % diffsonVersion % "test"
-    )
+    ),
+    // 2.x support was actually introduced in 1.3.0, but we forgot to publish the artifacts in later versions
+    tlVersionIntroduced := Map("3" -> "1.5.1", "2.13" -> "1.5.1", "2.12" -> "1.5.1")
```

Comment:
You can also remove specific missing values from `tlPreviousVersions` instead of just starting from 1.5.1.